src/core/sort.js

```diff
@@ -31,7 +31,8 @@ const invalidUnitParam = invalidParam(ES_REF_URL, 'unit', UNIT_SET);
  *     .query(esb.termQuery('user', 'kimchy'))
  *     .sort(esb.sort('post_date', 'asc'))
  *
- * @param {string} field The field to sort on
+ * @param {string=} field The field to sort on.
+ * If `.script()` function is used, It will be omitted.
```

Comment:
This could be reworded as
```suggestion
 * If a script is used to specify the sort order, `field` should be omitted.
```

Could you please do these changes in the 3 places? _I apologise for the redundancy, I am planning to do something about it sometime soon._

Reply:
That's so easy. 😄

Reply:
I agree that it's easy but it's also easy to miss and not obvious that the same changes have to made in multiple places. Nevertheless, I appreciate your patience and cooperation.